src/App.tsx

```diff
@@ -1,13 +1,9 @@
-
-
-
+import { Link } from "react-router-dom";
 export default function App() {
-  
-
   return (
+    // reroute the user to "/:room" when they click the btn (mefdev)
```

Comment:
Also, your GitHub username is logged in GitHub. You don't need to add your username `(mefdev)`